core/src/main/scala/cats/data/EitherT.scala

```diff
@@ -483,9 +483,13 @@ private[data] abstract class EitherTInstances1 extends EitherTInstances2 {
     new EitherTBitraverse[F] {
       val F0: Traverse[F] = F
     }
+
+  implicit def catsDataMonadErrorFForEitherT[F[_], E, L](implicit FE0: MonadError[F, E]): MonadError[EitherT[F, L, ?], E] =
```

Comment:
nitpick: would you add a comment doc here to indicate that this instance delegate to the `MonadError` of `F` instead of using `Either`?